### DISCUSSION TITLE
Allow to append to upload queue

### DISCUSSION THREAD

**Discussion by raphaelm** [ente-io/ente#3480] [Enhancements]
On Ente desktop, while uploading pictures, it is not possible to add more pictures to upload by either button press or drag and drop. I see no reason why they wouldn't just be added to the queue.

Backstory: I'm currently trying to upload my collection over a slow internet link. We're talking about a two-week task here ;) I upload a bunch of folders at a time. When I go to bed and the progress says "current upload almost finished", I have no way to add more pictures so the upload continues over night, before the current upload is fully finished. This makes this process even more tedious.

**Comment by mnvr** [comment]
In case it helps meanwhile - https://help.ente.io/photos/features/watch-folders

**Reply by raphaelm** [reply]
Another workaround I just found is opening multiple tabs of the web app ;)
